extensions/adhan-time/package.json

```diff
@@ -16,14 +19,14 @@
       "title": "Search Prayer Time",
       "description": "View adhan prayer times directly from Raycast",
       "mode": "view",
-      "interval": "24h"
+      "interval": "2h"
```

Comment:
why did you change this? and what is a 2h interval?

Reply:
I think I changed it while testing, will revert it back 👍🏼

Reply:
@mzaien I reverted it back, thanks for noticing 🙌🏼